stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -41,8 +41,9 @@ def Availability(bits):
 
 if Self == 'Float16':
     SelfDocComment = '''\
-/// A half-precision (16b), floating-point value type.'''
-
+/// A half-precision (16b), floating-point value type.
+///
+/// `Float16` is available on Apple silicon, and unavailable on Intel.'''
```

Comment:
Need a "when targeting macOS" here; it _is_ available on Intel/Linux and Windows.

Reply:
Changed in 4d35aae50e8.